randovania/games/dread/json_data/header.json

```diff
@@ -1200,6 +1200,11 @@
                 "long_name": "Ledge Warp",
                 "description": "A frame perfect trick that allows you to warp to a ledge you have previously been at by entering Morph Ball during the mantle animation. See https://youtu.be/3QE0tEC2tgc",
                 "extra": {}
+            },
+            "CBL": {
+                "long_name": "Cross Bomb Launch",
+                "description": "Sliding and morphing into an exploding cross bomb gives Samus a large amount of horizontal velocity.",
```

Comment:
I think I would have phrased this as "high horizontal velocity". I don't know why, but some reason, taking about an amount of velocity seems weird to me. Might just be a high level of pedantry at this point.

Reply:
Updated to "high horizontal velocity". From a mathy perspective, one can add or subtract varying amounts of velocity. I've not strong preference on the wording.